### PR TITLE
Add PSC05 test

### PR DESCRIPTION
**Pull request by tlawrencech**

### CODE REVIEW

File: cypress/integration/change_psc/change_legal_entity_psc.spec.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+const allForms = new AllFormsPage();
+const appointPSC02Page = new PscAppointment();
+const addressPage = new AddressPage();
+const preFilingPage = new PreFilingPSCPage();
+const pscLandingPage = new PSCLandingPage();
+const changeLegalEntityPsc = new ChangeLegalEntityPsc();
+
+
+describe('Change of a relevant legal entity with significant control (PSC) details - PSC05', () => {
+    beforeEach(() => {
```
Comment (rsuller) [inline cypress/integration/change_psc/change_legal_entity_psc.spec.js]: I've run this a couple of time, as part of the test suite and as a single spec. I've had different results. Can you try the same please to see what you get?
We may need to add some assertive step(s) in this `beforeEach` method to ensure we've achieved the correct state before running test.

File: cypress/integration/change_psc/change_legal_entity_psc.spec.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        changeLegalEntityPsc.changeEntityDetails();
+        cy.accessibilityCheck();
+        changeLegalEntityPsc.cancelEntityDetailsChange();
+
+        changeLegalEntityPsc.changeNatureOfControl();
+        cy.accessibilityCheck();
+        changeLegalEntityPsc.cancelNatureOfControlChange();
+
+         // Date of change and register entry date sections are already expanded
+         cy.accessibilityCheck();
```
Comment (rsuller) [inline cypress/integration/change_psc/change_legal_entity_psc.spec.js]: Format looks slightly out. Use `Shift+Alt+F` to quickly format files.

Comment (tlawrencech) [reply cypress/integration/change_psc/change_legal_entity_psc.spec.js]: Formatted

File: cypress/integration/change_psc/change_legal_entity_psc.spec.js (hunk 1 above)
Comment (tlawrencech) [reply cypress/integration/change_psc/change_legal_entity_psc.spec.js]: Added an assertion to check the confirmation of submission screen is displayed and that the PSC name is displayed in the list.  

This is not perfect by any means as we have still got the issue that webfiling may not actually appoint the psc02, but this is separate issue.

**Pull request merged**